src/content/plugins/split-chunks-plugin.mdx

```diff
@@ -147,7 +147,7 @@ module.exports = {
 };
 ```
 
-T> You can combine this configuration with the [HtmlWebpackPlugin](/plugins/html-webpack-plugin/). It will inject all the generated vendor chunks for you.
+T> You can combine this configuration with the [HtmlWebpackPlugin](/plugins/html-webpack-plugin/) for Single Page Application or [ChunksWebpackPlugin](/plugins/chunks-webpack-plugin/) for Multiple Page Application. It will inject all the generated vendor chunks for you.
```

Comment:
Can we just link to the github readme docs instead of adding it on webpack.js.org, since it is a 3rd party plugin?
```suggestion
T> You can combine this configuration with the [HtmlWebpackPlugin](/plugins/html-webpack-plugin/) for Single Page Application or [ChunksWebpackPlugin](https://github.com/yoriiis/chunks-webpack-plugin?tab=readme-ov-file#chunkswebpackplugin) for Multiple Page Application. It will inject all the generated vendor chunks for you.
```

Reply:
Yes of course, I've applied the suggestion